src/pipecat/services/fireworks.py

```diff
@@ -21,8 +21,9 @@
 class FireworksLLMService(BaseOpenAILLMService):
     def __init__(
         self,
+        api_key: str,
         *,
         model: str = "accounts/fireworks/models/firefunction-v1",
         base_url: str = "https://api.fireworks.ai/inference/v1",
     ):
-        super().__init__(model=model, base_url=base_url)
+        super().__init__(api_key=api_key, model=model, base_url=base_url)
```

Comment:
Adding this because otherwise it looks for `OPENAI_API_KEY` in the environment and that doesn't work for fireworks